tests/test_fields.py

```diff
@@ -571,3 +572,21 @@ def test_to_python(self):
 
         self.field.to_python({'raz': 1, 'dwa': 2})
         self.field.to_python('{"raz": 1, "dwa": 2}')
+
+
+class GeoPointTestCase(BaseTestCase):
+    field_name = 'geo_field'
+
+    def test_validate(self):
+
+        with self.assertRaises(SyncanoValueError):
+            self.field.validate(12, self.instance)
+
+        self.field.validate(models.GeoPoint.GeoPointStruct(**{'latitude': 52.12, 'longitude': 12.02}), self.instance)
```

Comment:
why not `GeoPointStruct(52.12, 12.02)` or GeoPointStruct(latitude=52.12, longitude=12.02) ?